### DISCUSSION TITLE
Change Nova Path in `config/nova.php` cant access the pages anymore.

### DISCUSSION THREAD

**Discussion by JenuelDev** [laravel/nova-issues#3346] [Q&A]
- Laravel Version: 8.12
- Nova Version: 3.22.0
- PHP Version: 7.4.13
- Database Driver & Version:
- Operating System and Version: Windows 10
- Browser type and version: Chrome

### Description:
After changing the path in `config/nova.php` I cant access pages inside the nova dashboard.

### Detailed steps to reproduce the issue on a fresh Nova installation:
First I went to `config/nova.php` and change the path from `nova` to `admin`
```php
'path' => '/admin',
```

after that, when I go to my project, `localhost:8000/admin`
the pages doesn't show up when path: 'localhost:8000/admin'
![image](https://user-images.githubusercontent.com/31676163/114815259-71135d00-9de8-11eb-9049-0e7372375700.png)

this is the original when the path is nova:
![image](https://user-images.githubusercontent.com/31676163/114815413-b46dcb80-9de8-11eb-99a9-13f70db674ec.png)

I need help changing the path from `nova` to `admin`

**Comment by crynobone** [comment]
![image](https://user-images.githubusercontent.com/172966/114822602-e7b65780-9df4-11eb-980a-81099562e130.png)

Unable to replicate, moving this to Discussion.

**Reply by JenuelDev** [reply]
hmm,,, maybe this happens in an existing project

**Reply by JenuelDev** [reply]
I just want to change path of nova to admin,, in an existing project

**Reply by crynobone** [reply]
Changing existing application should also work. There's no bug with Nova and you should consider checking `route:list` or `config:clear` etc

**Reply by JenuelDev** [reply]
if I change the path in `config/nova.php` to admin. I don't see any content on any page, and this error shows up in my console.
![image](https://user-images.githubusercontent.com/31676163/114845228-2efd1200-9e0e-11eb-91e1-abc7d1e1bf35.png)
